src/Symfony/Component/ErrorHandler/ErrorRenderer/HtmlErrorRenderer.php

```diff
@@ -23,6 +23,16 @@
  */
 class HtmlErrorRenderer implements ErrorRendererInterface
 {
+    public const IDE_LINK_FORMATS = [
```

Comment:
sorry, one more thought:
what about moving the const to the interface?

Reply:
Done.